controllers/validator.go

```diff
@@ -64,29 +61,14 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 	}
 
 	// check if the VSM plugin is specified or not
-	VSMPluginPresent := false
 	for _, plugin := range dpa.Spec.Configuration.Velero.DefaultPlugins {
 		if plugin == oadpv1alpha1.DefaultPluginVSM {
-			VSMPluginPresent = true
-		}
-	}
-
-	if r.checkIfDataMoverIsEnabled(&dpa) {
-		// parse for timeout if specified and see if there are no errors
-		if len(dpa.Spec.Features.DataMover.Timeout) > 0 {
-			_, err := time.ParseDuration(dpa.Spec.Features.DataMover.Timeout)
-			if err != nil {
-				return false, err
-			}
-		}
-
-		if !VSMPluginPresent {
-			return false, errors.New("datamover is enabled, specify vsm as a default plugin")
+			return false, errors.New("Delete vsm from spec.configuration.velero.defaultPlugins, it is not needed for DataMover usage in OADP 1.3")
 		}
 	}
 
-	if !r.checkIfDataMoverIsEnabled(&dpa) && VSMPluginPresent {
-		return false, errors.New("datamover is disabled, remove vsm as a default plugin")
+	if dpa.Spec.Features != nil && dpa.Spec.Features.DataMover != nil {
+		return false, errors.New("Delete dataMover object from spec.features, it is not needed for DataMover usage in OADP 1.3")
```

Comment:
It would be great to raise this reconcile error to the scrum team for review

Reply:
How url looks like in UI
![Screenshot from 2023-10-24 11-38-33](https://github.com/openshift/oadp-operator/assets/66965232/fc9dd9c7-b977-4fb1-a55a-bf898b7b8207)

Reply:
Going to update message to have both information in next commit